labellab-flask/api/middleware/logs_decorator.py

```diff
@@ -169,9 +170,23 @@ def wrap(*args, **kwargs):
         project_id = issue['project_id']
         project = find_by_project_id(project_id)
         message = f'{user["username"]} posted a new comment on {project["project_name"]}'
-        category = 'comments'
-        
+        category = 'issues'
```

Comment:
Also, can you make one small change here. The message should use issue title instead of project name.
Check #646 . Just make the same change. Then I won't need to merge that PR too.